packages/mui-joy/src/Select/Select.test.tsx

```diff
@@ -312,24 +312,12 @@ describe('Joy <Select />', () => {
       expect(getByRole('combobox')).not.to.have.attribute('aria-disabled');
     });
 
-    it('indicates that activating the button displays a listbox', () => {
-      const { getByRole } = render(<Select value="" />);
-
-      expect(getByRole('combobox')).to.have.attribute('aria-haspopup', 'listbox');
-    });
-
     it('renders an element with listbox behavior', () => {
       const { getByRole } = render(<Select defaultListboxOpen value="" />);
 
       expect(getByRole('listbox')).toBeVisible();
     });
 
-    specify('the listbox is automatically focused on open', () => {
```

Comment:
It's not focused on open anymore to prevent stealing focus from other elements on the page.